Regression on ROS 2 Buildfarm for Rolling

### Description

`moveit_setup_framework 2.13.0` is failing to build on the ROS 2 buildfarm for Rolling.
 

### ROS Distro

Rolling

### OS and version

24.04

### Source or binary build?

Binary

### If binary, which release version?

2.13.0

### If source, which branch?

_No response_

### Which RMW are you using?

None

### Steps to Reproduce

See https://build.ros2.org/view/Rbin_uN64/job/Rbin_uN64__moveit_setup_framework__ubuntu_noble_amd64__binary/140/consoleFull

### Expected behavior

Build succeeds. 

### Actual behavior

```
23:15:40 CMake Error at CMakeLists.txt:9 (find_package):
23:15:40   By not providing "Findament_cmake_ros.cmake" in CMAKE_MODULE_PATH this
23:15:40   project has asked CMake to find a package configuration file provided by
23:15:40   "ament_cmake_ros", but CMake did not find one.
23:15:40 
23:15:40   Could not find a package configuration file provided by "ament_cmake_ros"
23:15:40   with any of the following names:
23:15:40 
23:15:40     ament_cmake_rosConfig.cmake
23:15:40     ament_cmake_ros-config.cmake
23:15:40 
23:15:40   Add the installation prefix of "ament_cmake_ros" to CMAKE_PREFIX_PATH or
23:15:40   set "ament_cmake_ros_DIR" to a directory containing one of the above files.
23:15:40   If "ament_cmake_ros" provides a separate development package or SDK, be
23:15:40   sure it has been installed.
```

### Backtrace or Console output

_No response_

--- a/moveit_setup_assistant/moveit_setup_controllers/package.xml
+++ b/moveit_setup_assistant/moveit_setup_controllers/package.xml
@@ -8,7 +8,7 @@
   <license>BSD-3-Clause</license>
   <author email="davidvlu@gmail.com">David V. Lu!!</author>
 
-  <buildtool_depend>ament_cmake</buildtool_depend>
+  <buildtool_depend>ament_cmake_ros</buildtool_depend>
 
   <depend>ament_index_cpp</depend>
   <depend>moveit_setup_framework</depend>

--- a/moveit_setup_assistant/moveit_setup_core_plugins/package.xml
+++ b/moveit_setup_assistant/moveit_setup_core_plugins/package.xml
@@ -8,7 +8,7 @@
   <license>BSD-3-Clause</license>
   <author email="davidvlu@gmail.com">David V. Lu!!</author>
 
-  <buildtool_depend>ament_cmake</buildtool_depend>
+  <buildtool_depend>ament_cmake_ros</buildtool_depend>
 
   <depend>ament_index_cpp</depend>
   <depend>moveit_ros_visualization</depend>

--- a/moveit_setup_assistant/moveit_setup_framework/package.xml
+++ b/moveit_setup_assistant/moveit_setup_framework/package.xml
@@ -8,7 +8,7 @@
   <license>BSD-3-Clause</license>
   <author email="davidvlu@gmail.com">David V. Lu!!</author>
 
-  <buildtool_depend>ament_cmake</buildtool_depend>
+  <buildtool_depend>ament_cmake_ros</buildtool_depend>
   <depend>ament_index_cpp</depend>
   <depend>moveit_common</depend>
   <depend>moveit_core</depend>

--- a/moveit_setup_assistant/moveit_setup_simulation/package.xml
+++ b/moveit_setup_assistant/moveit_setup_simulation/package.xml
@@ -8,7 +8,7 @@
   <license>BSD-3-Clause</license>
   <author email="davidvlu@gmail.com">David V. Lu!!</author>
 
-  <buildtool_depend>ament_cmake</buildtool_depend>
+  <buildtool_depend>ament_cmake_ros</buildtool_depend>
 
   <depend>ament_index_cpp</depend>
   <depend>moveit_setup_framework</depend>

--- a/moveit_setup_assistant/moveit_setup_srdf_plugins/package.xml
+++ b/moveit_setup_assistant/moveit_setup_srdf_plugins/package.xml
@@ -8,7 +8,7 @@
   <license>BSD-3-Clause</license>
   <author email="davidvlu@gmail.com">David V. Lu!!</author>
 
-  <buildtool_depend>ament_cmake</buildtool_depend>
+  <buildtool_depend>ament_cmake_ros</buildtool_depend>
 
   <depend>moveit_setup_framework</depend>
   <depend>pluginlib</depend>

